functions/completeness.R

```diff
@@ -20,64 +18,54 @@ completeness <- function(end_date, level = c("board", "scotland")) {
     dplyr::filter(hb2014qf != "x") %>%
     dplyr::select(hb2014, hb2014name)
   
-  comp <- ckanr::ckan_fetch(paste0("https://www.opendata.nhs.scot/dataset/",
-                                   "110c4981-bbcc-4dcb-b558-5230ffd92e81/",
-                                   "resource/",
-                                   "daf55fd2-457f-4845-9af1-5d154cc0b19c/",
-                                   "download/financialyr.csv")) %>%
+  qtr <- ckanr::ckan_fetch(paste0("https://www.opendata.nhs.scot/dataset/",
+                                  "110c4981-bbcc-4dcb-b558-5230ffd92e81/",
+                                  "resource/03cf3cb7-41cc-4984-bff6-",
+                                  "bbccd5957679/download/quarters.csv")) %>%
     janitor::clean_names() %>%
+    dplyr::mutate(quarter = case_when(
+      str_detect(quarter, "Q1") ~ str_replace(quarter, "Q1", "Q4"),
+      str_detect(quarter, "Q2") ~ str_replace(quarter, "Q2", "Q1"),
+      str_detect(quarter, "Q3") ~ str_replace(quarter, "Q3", "Q2"),
+      str_detect(quarter, "Q4") ~ str_replace(quarter, "Q4", "Q3")
+    )) %>%
+    dplyr::mutate(quarter = 
+                    if_else(substr(quarter, 5, 6) == "Q4",
+                            paste0(parse_number(quarter) - 1, "Q4"),
+                            quarter)) %>%
     dplyr::filter(smr_type == "SMR01",
-                  readr::parse_number(financial_year) == 
+                  readr::parse_number(quarter) == 
                     lubridate::year(end_date) - 1) %>%
     dplyr::left_join(lookup, by = "hb2014") %>%
     dplyr::rename(board = hb2014name) %>%
     dplyr::mutate(board = replace(board,
                                   hb2014 == "S92000003",
                                   "Scotland")) %>%
-    tidyr::drop_na(board)
+    tidyr::drop_na(board) %>%
```

Comment:
I noticed you excluded SB0801 (The Golden Jubilee National Hospital). The special board lookup can be found at https://www.opendata.nhs.scot/dataset/non-standard-geography-codes-and-labels/resource/0450a5a2-f600-4569-a9ae-5d6317141899 

You have this hospital in the publication report. So we should have "The Golden Jubilee National Hospital" in "board" before drop_na(board). Need to do the same thing for fy.

Reply:
Thanks for spotting Tina - I've added this now. I did have a problem reading the special health board lookup using `ckanr::ckan_fetch()` so I've switched to using `read_csv` for all.